package excepciones;

public class Main
{
	public static void main(String[] args)
	{
		int[] numeros = new int[10];

		try
		{
			System.out.println("Hola");
			numeros[100000] = 4;
			System.out.println("Adi√≥s");
		}
		catch(ArrayIndexOutOfBoundsException e)
		{

			System.out.println("Error:" + e.getMessage());
		}
	}
}
